strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/storage/repository/Repository.java

```diff
@@ -359,4 +344,75 @@ public void setHttpConnectionPool(HttpConnectionPool httpConnectionPool)
     {
         this.httpConnectionPool = httpConnectionPool;
     }
+
+    public String getDetachAll()
+    {
+        return detachAll;
+    }
+
+    public void setDetachAll(String detachAll)
+    {
+        this.detachAll = detachAll;
+    }
+
+    public Long getVersion()
+    {
+        return version;
+    }
+
+    public void setVersion(Long version)
+    {
+        this.version = version;
+    }
+
+    public boolean isAllowsForceDeletion()
```

Comment:
We can’t do it that way because such kind of names breaks JavaBeans Naming Conventions that’s are part of java.io.Serializable requirements that needed to store entities in the OrientDB properly
(of course, keeping in mind current simplifications with BinaryConfiguration)